app/views/admin_general/index.html.erb

```diff
@@ -17,282 +17,204 @@
+              <%= @body_update_requests.size %>
+            </span>
+            <%= chevron_right %> Update authorities
+          </a>
+        </div>
+
+        <div id="update-authorities" class="accordion-body collapse">
+          <% for @change_request in @body_update_requests %>
+            <%= render :partial => 'change_request_summary' %>
+
+            <%= form_tag admin_change_request_path(@change_request),
+              :class => "form form-horizontal",
+              :method => 'put' do %>
+
+              <%= submit_tag 'Close', :class => "btn btn-danger" %>
+
+              <%= link_to "Close and respond",
+                edit_admin_change_request_path(@change_request),
+                :class => 'btn' %>
+
+              <%= link_to "Make update",
+                edit_admin_body_path(@change_request.public_body,
+                                     :change_request_id => @change_request.id),
+                                     :class => 'btn btn-primary' %>
+             <% end %>
+           <% end %>
+        </div>
+      </div>
+    <% end %>
+  </div>
+<% end %>
 
-        </table>
-      </div>
+<% if @comment_tasks %>
+  <div class="row">
+    <div class="span12">
+      <h2>Comments</h2>
     </div>
-  <% end %>
-
-  <% if @new_body_requests.size > 0 %>
-    <div class="accordion-group">
-      <div class="accordion-heading">
-        <a class="accordion-toggle" href="#new-authorities" data-toggle="collapse" data-parent="things-to-do">
-          <span class="label label-important">
-            <%= @new_body_requests.size %>
-          </span>
-          <%= chevron_right %> Add new authorities
-        </a>
-      </div>
-
-      <div id="new-authorities" class="accordion-body collapse">
-        <% for @change_request in @new_body_requests %>
-          <%= render :partial => 'change_request_summary'%>
-
-          <%= form_tag admin_change_request_path(@change_request),
-            :method => 'put',
-            :class => "form form-horizontal" do %>
-
-            <%= submit_tag 'Close', :class => "btn btn-danger" %>
-
-            <%= link_to "Close and respond",
-              edit_admin_change_request_path(@change_request),
-              :class => 'btn' %>
+  </div>
+  <div class="accordion" id="comments-things-to-do">
+    <%= render partial: 'to_do_list',
+               locals: { id: 'attention-comments',
+                         parent: 'comments-things-to-do',
+                         items: @attention_comments,
+                         label: 'Review comments reported by users
+                                 as requiring administrator attention' } %>
+  </div>
+<% end %>
 
-            <%= link_to "Add authority",
-              new_admin_body_path(:change_request_id => @change_request.id),
-              :class => 'btn btn-primary' %>
-          <% end %>
-        <% end %>
-      </div>
+<% if can?(:admin, AlaveteliPro::Embargo) && @embargoed_request_tasks %>
```

Comment:
Is this extra `can?` check necessary? We only set `@embargoed_request_tasks` if the user can admin the right things in the controller.